src/analysis.ts

```diff
@@ -0,0 +1,13 @@
+import {DocumentDescriptor} from './ast/ast';
+import {SerializedAnalysis} from './serialized-analysis';
+
+export class Analysis {
```

Comment:
Seems unnecessarily wordy. The result of the analyzer is naturally an analysis.

OTOH `breakfast food` is an actual phrase people use, so, I guess there's precedent.